include/kvdk/engine.h

```diff
@@ -103,18 +109,40 @@ extern KVDK_LIBRARY_API KVDKStatus KVDKHashGet(KVDKEngine *engine,
                                                const char *key, size_t key_len,
                                                size_t *val_len, char **val);
 
+// For Queue
+extern KVDK_LIBRARY_API KVDKStatus KVDKLPush(KVDKEngine *engine,
+                                             const char *collection,
+                                             size_t collection_len,
+                                             const char *key, size_t key_len);
+extern KVDK_LIBRARY_API KVDKStatus KVDKLPop(KVDKEngine *engine,
```

Comment:
Maybe we should use a String struct to pack these parameters.

Reply:
Aligned c++ API. Talked with @KVKIT , the current API maybe be more clear.